doc/development/style-guides.md

```diff
@@ -50,3 +59,48 @@ solgraph Market.sol > Market.dot
 brew install graphviz
 dot -Tpng Market.dot -o Market.png
 ```
+
+## TypeScript
+
+TypeScript code is linted with [tslint](https://palantir.github.io/tslint/) and its recommended rules. A sample `tslint.json` file would be:
+
+```json
+{
+  "defaultSeverity": "error",
+  "extends": [
+    "tslint:recommended"
+  ],
+  "jsRules": {},
+  "rules": {
```

Comment:
could we make it consistent with `eslint-config-oceanprotocol`? At least leaving out the semicolons? https://palantir.github.io/tslint/rules/semicolon/

Reply:
done